File: DSA/Raima_Rekhi_590013023/Day16Question2.c
#include <stdio.h>

int count_frequency(int arr[],int n,int element);

int main(){
int n,element;

printf("Enter the size of the array:\n");
scanf("%d",&n);
int arr[n];

// Prompt the user to enter the elements of the array
printf("Enter the elements of the array:\n");
for(int i=0;i<n;++i){
scanf("%d",&arr[i]);
}

// Prompt the user to enter the element whose frequency needs to be counted
printf("Enter the target element:\n");
scanf("%d",&element);

int count=count_frequency(arr,n,element);

if(count!=0)
printf("The frequency of the target element %d is: %d\n",element,count);
else
printf("No such element found.\n");
return 0;
}

// Function to find the frequency of an element in an array
int count_frequency(int arr[],int n,int element){
int count=0;
for(int i=0;i<n;++i){
if(arr[i]==element)
++count;
}
return count;
}

File: DSA/Nivedan_590014904/Day_16_Question_2.py
arr = []
n = int(input("Enter the number of elements in array:- "))
for i in range(n):
    val = int(input(f"Enter the {i} element: "))
    arr.append(val)
target = int(input("Target: "))
counter = 0
for i in range(n):
    if arr[i] == target :
        counter += 1
if counter > 0 :
    print(counter)
else:
    print(-1)
